test/e2e/storage/testsuites/volumemode.go

```diff
@@ -292,7 +298,15 @@ func (t *volumeModeTestSuite) DefineTests(driver TestDriver, pattern testpattern
 
 		ginkgo.By("Creating pod")
 		var err error
-		pod := e2epod.MakeSecPod(l.ns.Name, []*v1.PersistentVolumeClaim{l.Pvc}, nil, false, "", false, false, e2epv.SELinuxLabel, nil)
+		podConfig := e2epod.Config{
+			Ns:            l.ns.Name,
+			Pvclaims:      []*v1.PersistentVolumeClaim{l.Pvc},
+			SeLinuxLabel:  e2epv.SELinuxLabel,
+			NodeSelection: e2epod.NodeSelection{},
+		}
+		pod, err := e2epod.MakeSecPod(&podConfig)
+		framework.ExpectNoError(err, "Failed to create pod")
```

Comment:
This `MakeSecPod()` doesn't create a pod actually, it just returns a pod definition.
So it is better to change this error message `Failed to create pod` or just remove the message like the other error checks to avoid confusions later.